index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
We must either consider `new Boolean(false) === false` or consider them to be different types and not comparable. My implementations of `lte` return false if the values passed are of different types.